auditlog/conf.py

```diff
@@ -60,3 +60,7 @@
 settings.AUDITLOG_STORE_JSON_CHANGES = getattr(
     settings, "AUDITLOG_STORE_JSON_CHANGES", False
 )
+
+settings.AUDITLOG_DEFAULT_MASK_CALLABLE = getattr(
```

Comment:
Let's call it `AUDITLOG_MASK_CALLABLE`, because the default one is `mask_str`

Reply:
thank you! fixed. - 7b66442 @hramezani